notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -83,7 +83,9 @@
     "import hera_pspec as hp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def main_beam_FR_filter(data, wgts, tslice=tslice, cache={}, bls=None, suppression_factors=[FR_EIGENVAL_CUTOFF]):
</pre><p>Do we need this change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Debugging relic.